cluster_experiments/experiment_analysis.py

```diff
@@ -301,6 +302,112 @@ def from_config(cls, config):
         )
 
 
+class PairedTTestClusteredAnalysis(ExperimentAnalysis):
+    """
+    Class to run paired T-test analysis on aggregated data
+
+    Arguments:
+        cluster_cols: list of columns to use as clusters
+        target_col: name of the column containing the variable to measure
+        treatment_col: name of the column containing the treatment variable
+        treatment: name of the treatment to use as the treated group
+        strata_cols: list of index columns for paired t test. Should be a subset or equal to cluster_cols
+
+    Usage:
+
+    ```python
+    from cluster_experiments.experiment_analysis import PairedTTestClusteredAnalysis
+    from scipy.stats import  ttest_rel
+    import pandas as pd
+
+    df = pd.DataFrame({
+        'x': [1, 2, 3, 4, 0, 0, 1, 1],
+        'treatment': ["A", "B", "A", "B"] * 2,
+        'cluster': [1, 2, 3, 4, 1, 2, 3, 4],
+    })
+
+    PairedTTestClusteredAnalysis(
+        cluster_cols=df['cluster'],
```

Comment:
this should be 'cluster', not df['cluster']. I thought this was supposed to fail in unit tests, have you run pytest?

Reply:
I ran pytest (with df['cluster']) and it's passing. 

![image](https://user-images.githubusercontent.com/33734080/217016510-96759ec0-9d86-4a21-9148-b6fc584fbfff.png)

Will fix it anyway